style.css

```diff
@@ -70,7 +70,39 @@ header nav {
 header nav i {
   color: #fff;
   font-size: larger;
-  margin: px;
+}
+
+.navvme {
```

Comment:
- [ ] Great work implementing the mobile menu, however, the design is different from the Figma one. kindly adjust your design to be like the provided design as much as possible, including the fonts, margin/padding, and overall design

|Your design|Figma design|
|-|-|
|![Screenshot_20220620_224015](https://user-images.githubusercontent.com/25227022/174680789-c3ad9f89-1b7d-4076-8f4d-5e86ed9dc507.png)|![Screenshot_20220620_224138](https://user-images.githubusercontent.com/25227022/174680901-8ee22304-3624-4eea-afa9-d68974cdcdfa.png)|

Reply:
Design adjusted to be like the Figma design.